core/templates/pages/learner-dashboard-page/learner-dashboard-page.component.html

```diff
@@ -66,6 +66,11 @@ <h1 class="stat-value">{{ subscriptionsList && subscriptionsList.length }}</h1>
     <div class="col-md-2 oppia-learner-dashboard-side-content"
          [ngClass]="{'oppia-learner-dashboard-mobile-mode': checkMobileView()}">
       <mat-card class="oppia-learner-dashboard-main-menu">
+        <!-- <h4 class="oppia-learner-dashboard-menu"
```

Comment:
No, don't add commented out code. If this is not to be released now, create a flag and hide it behind an *ngIf on that flag.

Reply:
Done.